typescript/core/src/McapWriter.test.ts

```diff
@@ -344,4 +355,483 @@ describe("McapWriter", () => {
       },
     ]);
   });
+
+  it("supports append mode", async () => {
```

Comment:
this is a super long test. Maybe we could split them up in multiple `can append <schema,channel,message,...>`?

Reply:
I'm not sure if splitting them up would help since each of those tests would also be quite long (if we are testing the full file contents before & after appending). One option would be to move some of the expected values into a test fixtures file and import them, or use jest `expect.toMatchSnapshot` (so they are moved into a snap file). What do you think?

Reply:
Sounds like quite some work, I don't mind if it stays as it is